src/CLI/renderer/templates/show_tam_flow.j2

```diff
@@ -0,0 +1,39 @@
+{% set vars = {'name': ""} %}
+{% set vars = {'aclrule': ""} %}
+{% set vars = {'acltable': ""} %}
+{% set vars = {'samplingrate': ""} %}
+{% set vars = {'collector': ""} %}
+
+{% if json_output -%}
+{% for key_json in json_output %}
+    {% if key_json == "sonic-ifa:TAM_INT_IFA_FLOW_TABLE" %}
+        {% set flow_list = json_output[key_json]["TAM_INT_IFA_FLOW_TABLE_LIST"] %}
+    {% else %}
+        {% set flow_list = json_output[key_json] %}
+    {% endif %}
+{{'---------------------------------------------------'}}
+{{'TAM IFA Flow Information'.ljust(20)}}
+{% for flow in flow_list %}
+    {% if vars.update({'name':flow["name"]}) %}{% endif %}
+    {% if vars.update({'aclrule':flow["acl-rule-name"]}) %}{% endif %}
```

Comment:
@spenugondaa : Flow also needs to be displayed as Tables:
FLOW          ACL RULE            ACL TABLE          COLLECTOR           SAMPLING RATE

F1                     R1                        T1                        C1                                         10

F2                     R2                        T2                        C2                                        20

Reply:
Done